Infinum/ruleset.xml

```diff
@@ -74,12 +97,17 @@
 
   <!-- Exclude WordPress array indentation because it forced 4 space indentation -->
   <rule ref="WordPress.Arrays.MultipleStatementAlignment">
-    <exclude name="WordPress.Arrays.ArrayIndentation.ItemNotAligned" />
     <properties>
       <property name="alignMultilineItems" value="!=100"/>
     </properties>
   </rule>
 
+  <!-- Exclude the multiline array indentation sniff,
+  because it clashes with Generic indentation sniff -->
+  <rule ref="WordPress.Arrays.ArrayIndentation">
+    <severity>0</severity>
```

Comment:
So now we have no rules on array indentation?

Reply:
No, the `Generic.Arrays.ArrayIndent` takes care of array indentation :) WordPress rules are kinda quirky